www/favicons/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
+    "short_name": "",
+    "icons": [
+        {
+            "src": "/favicons/android-chrome-192x192.png",
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "/favicons/android-chrome-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
+    ],
+    "theme_color": "#ffffff",
+    "background_color": "#ffffff",
+    "display": "standalone"
```

Comment:
`"display": "standalone"` means "don't show the URL bar." IMO that's an unwise choice for any site that isn't designed to handle it. For example, there would be no obvious way to use the back button if you a2hs. I think `"display": "minimal-ui"` or `"display": "browser"` would be more appropriate.

Reply:
Got it, thanks! I confess I basically just followed their wizard and so didn't know what all of these stood for. Maybe just `"display": "browser"` for now since it appears to be straightforward and we can be reasonably certain it won't break anything.